libs/markets/src/lib/components/market-info/market-info-panels.tsx

```diff
@@ -659,6 +671,91 @@ export const LiquidityMonitoringParametersInfoPanel = ({
   return <MarketInfoTable data={marketData} parentData={parentMarketData} />;
 };
 
+export const EthOraclePanel = ({ sourceType }: { sourceType: EthCallSpec }) => {
+  const abis = sourceType.abi?.map((abi) => JSON.parse(abi));
+  const header = 'uppercase my-2 font-bold text-left';
+  return (
+    <>
+      <h3 className={header}>{t('Ethereum Oracle')}</h3>
+      {sourceType.address && (
+        <>
+          <KeyValueTable>
+            <KeyValueTableRow noBorder>
+              <div>{t('Address')}</div>
+              <CopyWithTooltip text={sourceType.address}>
+                <button
+                  data-testid="copy-eth-oracle-address"
+                  className="underline text-right"
+                >
+                  {truncateMiddle(sourceType.address)}
+                  <Icon name="duplicate" className="ml-2" />
+                </button>
+              </CopyWithTooltip>
+            </KeyValueTableRow>
+          </KeyValueTable>
+
+          <div className="my-2">
+            <EtherscanLink address={sourceType.address}>
+              {t('View on Etherscan')}
+            </EtherscanLink>
+          </div>
+        </>
+      )}
+
+      <MarketInfoTable
+        key="eth-call-spec"
+        data={{
+          method: sourceType.method,
+          requiredConfirmations: sourceType.requiredConfirmations,
+        }}
+      />
+      <Accordion>
+        <AccordionPanel
+          itemId="abi"
+          trigger={
+            <AccordionPrimitive.Trigger
+              data-testid="accordion-toggle"
+              className={classNames(
+                'w-full pt-2',
+                'flex items-center gap-2',
+                'group'
+              )}
+            >
+              <div
+                data-testid={`abi-dropdown`}
+                key={'value-dropdown'}
+                className="flex items-center gap-2 w-full"
+              >
+                <div className="uppercase font-bold"> {t('ABI')}</div>
+                <AccordionChevron size={14} />
+                <div className="flex items-center gap-1"></div>
+              </div>
+            </AccordionPrimitive.Trigger>
+          }
+        >
+          <SyntaxHighlighter data={abis} />
+        </AccordionPanel>
+      </Accordion>
+
+      <h3 className={header}>{t('Normalisers')}</h3>
+      {sourceType.normalisers?.map((normaliser, i) => (
+        <MarketInfoTable key={i} data={normaliser} />
+      ))}
+      <h1 className={header}>{t('Filter')}</h1>
+      <h1 className={header}>{t('Key')}</h1>
+      {sourceType.filters?.map((filter, i) => (
+        <>
+          <MarketInfoTable key={i} data={filter.key} />
+          <h1 className={header}>{t('Conditions')}</h1>
+          {filter.conditions?.map((condition, i) => (
```

Comment:
If there are multiple conditions I don't think its clear that this is a list. You end up with data labels with the same text level at the same level of hierarchy. I think it needs a more thought on how to lay this out.

Reply:
I don't know if we need to expose all these conditions yet. Will double-check.

Reply:
I have used the `ConditionOperatorMapping` please check the updated screenshots

Reply:
I was going to question the multiple `h3`s in a row too (especially as the first can be empty) but like @mattrussell36 that it can go in as-is and be fixed in a follow up